go.mod

```diff
@@ -1,6 +1,8 @@
 module zotregistry.io/zot
 
-go 1.20
+go 1.21
```

Comment:
@elee1766 did you need go 1.21? We are still migrating to that.
https://github.com/project-zot/zot/pull/2049

This new "toolchain" directive is breaking all CI tests.

Reply:
i don't think i need it. i'll push with 1.20